doc/release-notes/RELEASE-0.8.md

```diff
@@ -0,0 +1,59 @@
+
+# Release v0.8
+
+We are pleased to announce the release of Screed v0.8. Screed is a database
+engine capable of storing and retrieving short-read sequence data and is
+designed to be fast and adaptable to different sequence file formats.
+
+This version of Screed contains developer documentation for contributing to the
+Screed project and a code of conduct for interacting with other contributors
+and project maintainers. Documentation is available at
+http://screed.readthedocs.org/en/v0.8/
+
+## New items of note:
+
+This release successfully installs and passes its unit tests on
+Ubuntu 14.04 and the latest release of Mac OS X 10 "Yosemite". It
+also passes the khmer acceptance tests as per the [eelpond testing
+protocol.]
+(https://github.com/ged-lab/literate-resting/blob/master/kp/README.txt)
+
+This release of screed has renamed the 'accuracy' attribute of read records to
+'quality;' this API change will need to be adopted by all users wanting to
+upgrade to this version. Unlike the khmer project, Screed is not currently
+under semantic versioning. It will be with the 1.0 release.
+
+ - Screed now has automatic compression detection via magic bit sniffing for 
+ gzip, bzip files (from @mr-c in #432)
+ - Screed now supports streaming of uncompressed FASTA and FASTQ files. Bzip2
+ files can also be streamed but not gzip files. (from @mr-c, see #633)
```

Comment:
"BZIP2 compressed FASTA and FASTQ formatted nucleotide sequence data can also be streamed but not GZIP compressed FASTA and FASTQ formatted nucleotide sequence data."